projects/cdk/directives/swipe/swipe.service.ts

```diff
@@ -52,6 +54,7 @@ export class TuiSwipeService extends Observable<TuiSwipeEvent> {
                         return null;
                     }),
                     filter(tuiIsPresent),
+                    takeUntilDestroyed(destroyRef),
```

Comment:
Looks like we don't need to use destroyRef in observable constructor. It's enough to return subscription (as TearDown Logic)
same for pan service

Reply:
fixed